### DISCUSSION TITLE
cn/2018/03/manifold-learning/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#35] [Comments]
# cn/2018/03/manifold-learning/

降维 在之前的 博客 中，我们曾经介绍过 PCA 方法及其降维的作用。在原始数据中各个特征之间存在着一定的信息冗余，随着特征的不断增加就容易出现“维数灾难

https://leovan.me/cn/2018/03/manifold-learning/

**Comment by xiyuduann** [comment]
博主您好！最近在了解流形学习，看github上您分享的代码失效了，请问可以再分享一次吗？

**Reply by leovan** [reply]
链接已修复
